docs/servo_power_test/package.json

```diff
@@ -33,6 +33,6 @@
     "@types/node": "20.8.2"
   },
   "engines": {
-    "node": ">= 11.0.0"
+    "node": ">= 12.0.0"
```

Comment:
nit: is it intended? (just making sure...)

Reply:
Object.fromEntries() is supported from node.js 12.0.0, so updated node.js in package.json.

Reply:
Got it. Thanks for the explanation!!